packages/docs/examples/va-sidebar/Color.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
Should use VaList component and a bit of variety. We want examples to look similarly to standard components.